src/components/LineItems.jsx

```diff
@@ -1,5 +1,6 @@
 import AddCircleIcon from '@mui/icons-material/AddCircle';
 import { Grid, IconButton, Typography } from '@mui/material';
+import { ListItem, List } from '@mui/material'; //TODO MAKE SURE TO CHANGE THIS IMPORT
```

Comment:
one last thing, not exactly sure about the comment but these could be combined in import statement on line 2

Reply:
that's exactly why I left that comment lol. I separated them because I kept commenting them out to test without getting the unused-vars error